index.d.ts

```diff
@@ -1,8 +1,21 @@
 import {PackageJson} from 'type-fest';
-import {Options as MinimistOptions} from 'minimist-options';
 
 declare namespace meow {
-	interface Options<Flags extends MinimistOptions> {
+	type FlagType = 'string' | 'boolean' | 'number';
+
+	interface Flag<Type extends FlagType, Default> {
+		readonly type?: Type;
+		readonly alias?: string;
+		readonly default?: Default;
+	}
+
+	type StringFlag = Flag<'string', string>;
+	type BooleanFlag = Flag<'boolean', boolean>;
+	type NumberFlag = Flag<'number', number>;
+
+	type BaseFlags = { [key: string]: StringFlag | BooleanFlag | NumberFlag };
```

Comment:
I don't really like the name `BaseFlags`. Do you have another good name?

Reply:
`AnyFlags`?

Reply:
Looks nice! Thank you, I'll change.